Makefile

```diff
@@ -0,0 +1,101 @@
+BINARY_NAME := "wisee"
+DEV_DATABASE_URL := "postgres://postgres:postgres@localhost:5432/wisee_core?sslmode=disable"
```

Comment:
why ssl mode is disable../

Reply:
- as this will only be used for connecting to the db when in development, should I set it to `prefer` / `require`?

Reply:
> * as this will only be used for connecting to the db when in development, should I set it to `prefer` / `require`?

do you mean in prod this will be true..?